app/layout.tsx

```
@@ -46,11 +72,16 @@ export default async function RootLayout({
 }>) {
   const nonce = (await headers()).get('x-nonce') || undefined;
```

ESLint

[error] The value assigned to 'nonce' is not used in subsequent statements.

(no-useless-assignment)